.gitignore

```diff
@@ -51,3 +51,4 @@ flycheck-*
 
 ### vim
 .ropeproject/
+*~
```

Comment:
Please don't modify this file, instead update your local .gitignore to cover for those files.